kxml_compiler/creator.cpp

```diff
@@ -260,7 +260,8 @@ ClassDescription Creator::createClassDescription(const Schema::Element &element)
 
         QString targetClassName = Namer::getClassName(targetElement.name());
 
-        if (targetElement.text() && !targetElement.hasAttributeRelations() && !r.isList()) {
+        if ((targetElement.text() || targetElement.type() < Schema::Element::ComplexType)
```

Comment:
Why `<` and not just `!=`?

Reply:
> Why `<` and not just `!=`?

Makes sense I will fix it!

Reply:
This probably means there's no need for it to be last anymore, i.e. you could revert the change to the enum, unless I'm missing something?

Reply:
Yeah. When I started to work on this I was not 100% confident that no other kind of types needs to be treated similar to complexTypes. But at the end it turned out that no need for that.

Also it might be useful to remove/refactor the Schema::Element::text in this PR because  I think the name 'text' is a bit misleading and from the type (complexType or not) and from the number of attributes and relations it is possible to determine that a simple C++ type (QString/int/something similar) member or a class representing the questioned element shall be generated.